modules/git/commit.go

```diff
@@ -21,6 +21,7 @@ import (
 	"strings"
 
 	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/mcuadros/go-version"
```

Comment:
I'm not sure if that's the right package to use

Reply:
github.com/hashicorp/go-version is used elsewhere

Reply:
Fixed.